packages/wallet-lib/src/test/utils.js

```diff
@@ -0,0 +1,14 @@
+const sleepOneTick = () => new Promise((resolve) => {
+  if (typeof setImmediate === 'undefined') {
+    setTimeout(resolve, 10);
+  } else {
+    setImmediate(resolve);
+  }
+});
+
+const sleep = (timeout) => new Promise(((resolve) => setTimeout(resolve, timeout)));
+
+module.exports = {
+  sleepOneTick,
+  sleep,
```

Comment:
We call this function `wait` in all other packages. Should we keep the name consistent?

Reply:
Updated